### PR TITLE
Added YOU tag on your own comments.

### PR DESCRIPTION
**Pull request by mykdavies**
Looks quite nice.

### CODE REVIEW

File: lib/widgets/comment/comment.dart [hunk 1]
```diff
@@ -242,25 +246,29 @@ class _CommentWidget extends HookWidget {
...
+                                context.read<ConfigStore>().commentTitleSize,
+                            color: theme.colorScheme.secondary,
                           ),
                         ),
                       ),
                       if (creator.isCakeDay) const Text(' 🍰'),
+                      if (creator.id ==
+                          (_tryGetUserData(context, store.comment.instanceHost)
+                                  ?.userId ??
+                              -1))
```
Comment (shocklateboy92) [inline lib/widgets/comment/comment.dart:266]: Why is this `?? -1` needed? the `==` operator can compare with `null` just fine.

**Review by shocklateboy92 — COMMENTED** [review]
Not that I don't trust you when you say "it looks good", but a screenshot would be nice :slightly_smiling_face:

File: lib/widgets/comment/comment.dart (hunk 1 above)
Comment (mykdavies) [reply lib/widgets/comment/comment.dart:266]: Good point, thanks

**Comment by mykdavies** [comment]
> Not that I don't trust you when you say "it looks good", but a screenshot would be nice 🙂

I didn't even realise that I could add screenshots in! Here you go, light mode of course...
Screenshot 2023-07-12 at 02 57 13

**Review by zachatrocity — APPROVED** [review]
LGTM 👊

**Pull request merged**